testing/architecture-test/src/changes/accepted-changes/accepted-public-api-changes.json

```diff
@@ -248,14 +248,6 @@
                 "From non-null returning to null returning breaking change"
             ]
         },
-        {
-            "type": "org.gradle.api.file.RelativePath",
```

Comment:
Looks like we removed two entire from the accepted public api changes but I can't seem to find corresponding code changes in this pr.

What caused these?

Reply:
These are still removed?

Reply:
Yeah, at this point IDK what's up with these.  Rebasing keeps removing them, and it seems they're not actually needed, as `sanityCheck` continues to pass.